### ISSUE TITLE
plugin ts template: Problem with parameter properties

### ISSUE THREAD

**Issue by Franck-El** [jspsych/jsPsych#2672]
When using the jsPsych builder (4.3.0) and jsPsych (7.2), the template in TypeScript to make its own plugin needs this replacement to work :

```    
    // constructor(private jsPsych: JsPsych) { } // comment this line 

    // replace by this
    private jsPsych: JsPsych;

    constructor(jsPsych: JsPsych) {
        this.jsPsych = jsPsych;
    }
````
I'm relatively new to TS so can't understand why, but it is the case. To made a few tests I installed typescript -D and run `tsc` on those two versions of the file and, as I learned it, it was strictly equivalent. So I suppose this issue come from one parameter used in the tsconfig or wepback.config file of the builder. I could read them [here](https://github.com/bjoluc/jspsych-builder) to check but unforntunately my level doesn't allow me to find what can cause this problem.

**Comment by bjoluc** [comment]
Thanks @Franck-El for bringing this up. Unfortunately, I can't think of any typescript config option that would disable [parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties). The builder uses Babel and its TypeScript preset to handle ts files, so maybe it's an oddity with that setup. I'll update you after looking into this.

**Comment by bjoluc** [comment]
It was a Babel configuration issue indeed. I just released the fix in v4.3.1. Thanks!
